<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ActivityNetworkStatusClass</class>
 <widget class="QDialog" name="ActivityNetworkStatusClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>302</width>
    <height>442</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>About P2PThing</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog#ActivityAbout
{   
color: transparent;
background-color: transparent;
border: 1px solid transparent;
border-radius: 8px; 
}

#StyledDlgTitleBox
{   
color: black;
background-color: rgb(50,238,17);
border: 2px solid green;
border-top-left-radius: 8px; 
border-top-right-radius: 8px;
}

#StyledDlgTitleLabel
{   
color: black;
}

#StyledDlgItemsBox
{   
background-color: rgb(50,238,17);
border: 2px solid green;
border-top-color: green; 
border-bottom-left-radius: 8px; 
border-bottom-right-radius: 8px;
}
QPushButton#CancelButton
{
color: white;
font: bold 14px;
border-width: 10;
border-image: url(:/MyP2PWeb/Resources/button_green_normal.png) 10;
padding: -10;
}
QPushButton#CancelButton:pressed
{
border-width: 10;
border-image: url(:/MyP2PWeb/Resources/button_green_pressed.png) 10;
padding: -10;
}
QPushButton#AcceptButton
{
color: white;
font: bold 14px;
border-width: 10;
border-image: url(:/MyP2PWeb/Resources/button_green_normal.png) 10;
padding: -10;
}
QPushButton#AcceptButton:pressed
{
border-width: 10;
border-image: url(:/MyP2PWeb/Resources/button_green_pressed.png) 10;
padding: -10;
}
QLabel#PermissionLabel
{
font: bold 14px;
}
QLineEdit#FileNameEdit
{
background-color: rgb(238,238,238);
}
QPlainTextEdit#OfferMsgEdit</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="TitleBarWidget" name="m_TitleBarWidget" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>50</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="StyledDlgItemsBox">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="title">
      <string/>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="spacing">
       <number>5</number>
      </property>
      <property name="margin">
       <number>5</number>
      </property>
      <item>
       <widget class="QLabel" name="m_NetworkTypeTile">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Internet Connection</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="m_CellNetworkTitle">
          <property name="text">
           <string>Cellular Data Connected?:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="m_CellNetworkStateLabel">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>NO</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="m_InternetConnectedTitle">
          <property name="text">
           <string>Internet Connected?:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="m_InternetConnectedStateLabel">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>NO</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="m_IPAddressTitle">
          <property name="text">
           <string>IP Address:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="m_IPAddressValueLabel">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="m_TcpPortTitle">
          <property name="text">
           <string>TCP Port:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="m_TcpPortValueLabel">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="labelCopywrite2">
        <property name="text">
         <string>My P2P Web Network</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_7">
        <item>
         <widget class="QLabel" name="m_MyNodeUrlTitle">
          <property name="text">
           <string>My Node Url:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="m_MyNodeUrlValueLabel">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string notr="true"/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QLabel" name="m_P2PWebNetStateTitle">
          <property name="text">
           <string>Network State:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="m_P2PWebNetStateValueLabel">
          <property name="sizePolicy">
           <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Unknown</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>10</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="m_NetStatLine1">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="m_NetStatLine2">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="m_NetStatLine3">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="m_NetStatLine4">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="m_NetStatLine5">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="m_NetStatLine6">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="m_NetStatLine7">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="m_NetStatLine8">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="gotoWebsiteButton">
        <property name="text">
         <string>go to http://www.myp2pweb.com</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TitleBarWidget</class>
   <extends>QWidget</extends>
   <header>TitleBarWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
